src/sim-host/protocol/socket.js

```diff
@@ -49,6 +49,17 @@ module.exports.initialize = function (pluginHandlers, services) {
         document.location.reload(true);
     });
 
+    socket.on('retheme', function () {
+        var links = document.head.getElementsByTagName('link');
+        var themeLinks = Array.prototype.filter.call(links, function (elt){
+            return elt.href.indexOf("sim-host-theme.css") >= 0;
+        });
+        if (themeLinks.length > 0) {
+            // Trigger the script to reload
+            themeLinks[0].href = "sim-host-theme.css";
```

Comment:
Yes. It's actually not "the same value", since the `themeLinks[0].href` value is a fully qualified path, but empirically this works in chrome at least.